metatests.js

```diff
@@ -23,3 +23,19 @@ module.exports = {
   testAsync,
   walk,
 };
+
+if (process.browser) {
+  process.exit = code => {
+    /* eslint-disable no-undef*/
+    __karma__.info({ total: 1 });
+    __karma__.result({ success: !code });
+    __karma__.complete();
+    /* eslint-enable*/
+  };
+  metatests.runner.instance.on('finish', () => {
+    console.log('Tests finished. Waiting for unfinished tests after end...');
+    setTimeout(() => process.exit(0), 5000);
+  });
+}
```

Comment:
Also, what is it doing here? Shouldn't it be in `cli.js`?

Reply:
`cli.js` is running on the server side and this part of `metatests.js` - only on client side

Reply:
But why can't you put everything in your `process.exit` overrride to the `setTimeout` call later?

Reply:
With https://github.com/metarhia/metatests/pull/79 there won't be `process.exit()` at all.

Reply:
Ok, then. I thought you maybe missed it.